pkg/ibmvpc/nacl_analysis.go

```diff
@@ -74,15 +75,16 @@ func (na *IBMNACLAnalyzer) GetNACLRule(index int) (ruleStr string, ruleRes *comm
 		)
 		srcPorts := getPortsStr(*ruleObj.SourcePortMin, *ruleObj.SourcePortMax)
 		dstPorts := getPortsStr(*ruleObj.DestinationPortMin, *ruleObj.DestinationPortMax)
-		connStr = fmt.Sprintf("protocol: %s, srcPorts: %s, dstPorts: %s", *ruleObj.Protocol, srcPorts, dstPorts)
+		protocol = *ruleObj.Protocol
+		portsStr = fmt.Sprintf(", srcPorts: %s, dstPorts: %s", srcPorts, dstPorts)
 		direction = *ruleObj.Direction
 		src = *ruleObj.Source
 		dst = *ruleObj.Destination
 		action = *ruleObj.Action
 	case *vpc1.NetworkACLRuleItemNetworkACLRuleProtocolIcmp:
 		name = *ruleObj.Name
 		conns = commonvpc.GetICMPconn(ruleObj.Type, ruleObj.Code)
-		connStr = fmt.Sprintf("protocol: %s", *ruleObj.Protocol)
+		protocol = *ruleObj.Protocol
```

Comment:
shouldn't you also set `portsStr` to show code and type (if relevant)?

Reply:
Guess we should. see comment above.
I would change the code so that it uses the printing function of conn; not sure why it was not done this way in the first place; perhaps I am missing something.

Reply:
fixed, see comment above